### PR TITLE
[DATA-1377] Added a DAG view for related jobs

### PR DESCRIPTION
**Pull request by nand-stripe**
The DAG view displays a graph of dependencies between the mapreduce staps which are part of the scalding pipeline. The links are determined using the input/output paths of individual jobs.

screen shot 2017-01-20 at 12 54 03 pm

### CODE REVIEW

File: js/components/related-dag.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    />
+                );
+              })}
+              {g.nodes().map((key, i) => {
+                const node = g.node(key);
+                return (
+                    <g key={i} transform={`translate(${node.x}, ${node.y})`}>
+                      <circle
+                          r="20"
+                          style={{
```
Comment (abull-stripe) [inline js/components/related-dag.jsx]: I usually like making styles like this a variable outside the jsx

Comment (abull-stripe) [inline js/components/related-dag.jsx]: or outside whatever you call the xml formatted jsx

**Comment by abull-stripe** [comment]
do we have somewhere where we have the list of vendored js libs, or do we just drop them in the js/libs directory

**Comment by abull-stripe** [comment]
just minor nits

**Comment by nand-stripe** [comment]
It seems that there is no list of external libraries anywhere.

**Comment by abull-stripe** [comment]
okay cool, the list of vendored libs is small so I'm not too bothered

**Comment by abull-stripe** [comment]
splitting components lgtm

**Comment by abull-stripe** [comment]
lgtm

**Pull request merged**